src/components/svg/wrap/svg-keyboard-wrap.component.scss

```diff
@@ -0,0 +1,21 @@
+:host {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    display: block;
+}
+
+svg-keyboard {
+    width: 95%;
+    max-width: 1400px;
```

Comment:
This value is used to limit keyboard width on big screens. This is not a new code.